student.rb

```diff
@@ -3,14 +3,15 @@
 class Student < Person
   def initialize(age, classroom, name: 'Unknown', parent_permission: true)
     super(age, name: name, parent_permission: parent_permission)
```

Comment:
- [ ] I think you forgot to add attr_reader :classroom so that the Person file could be read.